### PR TITLE
bumped version changes

### PR DESCRIPTION
**Pull request by richagadgil**

### CODE REVIEW

File: src/defaults/tabSpace.tsx [hunk 1]
```diff
@@ -135,10 +135,7 @@ class TabSpace extends VDomRenderer<TabSpace.Model> implements ITabSpace {
         super.onUpdateRequest(msg);
     }
 
-    private _onActiveCellChange = (
-        _tracker: INotebookTracker,
-        cell: Cell | null
-    ) => {
+    private _onActiveCellChange = (_tracker: INotebookTracker, cell: Cell | null) => {
```
Comment (declanvk) [inline src/defaults/tabSpace.tsx]: Did the signature of these change?  I thought they converted to `(_tracker: INotebookTracker, cell: Cell) => void`. Also why is there the need to add all of these `!` overrides.

File: src/defaults/lineCol.tsx [hunk 2]
```diff
@@ -249,7 +249,7 @@ class LineCol extends VDomRenderer<LineCol.Model> implements ILineCol {
                 if (activeCell === undefined) {
```
Comment (declanvk) [inline src/defaults/lineCol.tsx]: Also this should be a test for `null` rather than `undefined`
